cmd/tools/rpcwrappers/main.go

```diff
@@ -93,9 +93,10 @@ var (
 		"retryableClient.history.StreamWorkflowReplicationMessages": true,
 
 		// these are non-standard implementations. do not generate.
-		"client.history.DescribeHistoryHost":    true,
-		"client.history.GetReplicationMessages": true,
-		"client.history.GetReplicationStatus":   true,
+		"client.history.DescribeHistoryHost":                    true,
+		"client.history.GetReplicationMessages":                 true,
+		"client.history.GetReplicationStatus":                   true,
+		"client.history.VerifyChildExecutionCompletionRecorded": true,
```

Comment:
nit: do you also need to add StreamWorkflowReplicationMessages?

Reply:
the override is on line # 91 https://github.com/temporalio/temporal/pull/4708/files#diff-224d80d92f0d36d3692f634944dc3462611a85efe56db8b8546f8662521308f2R91